msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -27,7 +27,7 @@ Tabs allow team members to access services and content in a specific space withi
 > If you have integrated a tab with your meeting, then your app must follow the Teams [single sign-on (SSO) authentication flow for tabs](../tabs/how-to/authentication/auth-aad-sso.md).
 
 > [!NOTE]
-> Apps are supported in private scheduled meetings only.
+> Apps are only supported in private scheduled meetings and standard channel meetings.
 
 #### Pre-meeting app experience
 
```

Comment:
For the pre-meeting app experience, can we note that only "Add" is supported here. Rename and remove can be done from meeting chat or in-meeting experience.

Reply:
Also, for Channel meetings, there is a limit of 5 tabs per meeting.